doc/subsystems/conf_s11n/conf_s11n.rst

```diff
@@ -0,0 +1,156 @@
+.. _conf_s11n
+
+Runtime configuration system
+############################
+
+Config subsystem is meant for persisting per-device configuration
```

Comment:
The config subsystem gives modules a way to store persistent
    per-device configuration and runtime state.